content/documentation/pages/6-recipes/7-scaling/2-autoscaling.md

```diff
@@ -0,0 +1,198 @@
+The `spring.cloud.stream.kafka.binder.minPartitionCount` property sets the minimum number of partitions that the Kafka binder configures on the topic, which is where the transform-processor is subscribing for new data.
+
+![SCDF autoscaling stream deploying](images/scdf-autoscaling-stream-deploying.png)
+
+Use the SCDF's built-in Grafana dashboard to review the [stream application's throughput and the other application metrics](%currentPath%/feature-guides/streams/monitoring/#prometheus-1):
+
+![SCDF autoscaling initial metrics](images/scdf-autoscaling-initial-metrics.png)
+
+The `time`, `transform` and `log` applications maintain the same message throughput (~1 msg/s). The `transform` copes fine with the current load.
+
+### Increase pipeline load
+
+Now, let's increase the load by increasing the time-source's message production rate. By changing time's `time-unit` property from `MILLISECONDS` to `MICROSECONDS` the input rate will increase from one to a couple of thousands of messages per second.
+Note that the [stream rolling-update](%currentPath%/stream-developer-guides/continuous-delivery/) functionality allows to rolling-update the time application without stopping the entire stream:
+
+```shell
+stream update --name scaletest --properties "app.time.trigger.time-unit=MICROSECONDS"
+```
+
+The `time` app is re-deployed with the new time-unit property:
+![SCDF autoscaling increase load](images/scdf-autoscaling-increase-load.png)
+
+Now `time` source emits messages with a rate of `~5000 msg/s`. The `transform` processor, though, is capped at around `1000 msg/s` and that in turn, it halts the throughput of the entire stream to a certain level. This is an indicator that the `transform` has become the bottleneck.
+
+![SCDF autoscaling pipeline bottleneck](images/scdf-autoscaling-pipeline-bottleneck.png)
+
+The `HighThroughputDifference` Prometheus alert rule detects the rate discrepancy and fires a scale-out alert:
+
+![SCDF autoscaling scaleout alert](images/scdf-autoscaling-scaleout-alert.png)
+
+in result 3 additional transform instances are added:
+
+![SCDF autoscaling adding 3 instances](images/scdf-autoscaling-adding-3-instances.png)
+
+with the help of the additional instance the `log` sink and the entire pipeline catch up match with the `time` source:
+
+![SCDF autoscaling stream catches up](images/scdf-autoscaling-stream-catchup.png)
+
+### Reduce pipeline load
+
+If we reduce the source's data production rate back to the original rate (i.e., `1 msg/s`):
+
+```shell
+stream update --name scaletest --properties "app.time.trigger.time-unit=MILLISECONDS"
+```
+
+The extra `transform` processor instances aren't altering the overall throughput rate anymore.
+Eventually the rate difference becomes zero and the `ZeroThroughputDifference` alert is fired. Later in turn triggers a scale-in action and the extra instances can be scaled back in:
```

Comment:
We could minimize the use of "Later" wherever possible. Maybe you're referring to "latter", but it is still a bit hard to grok when used in the sentence.

Reply:
yeh, i mean latter indeed.  But replaced it with `The alert in turn ...`